### ISSUE TITLE
gradle error

### ISSUE THREAD

**Issue by hamedisu** [aspose-cells/Aspose.Cells-for-Java#46]
hi
I used the aspose cell in form of maven and it works
but now I want to merge the project with another java project using gradle (instead of maven)
but after adding the jar files downloaded from your site (from this page: https://downloads.aspose.com/cells/java), I face with this error:
error: package com.aspose does not exist
import com.aspose.cells.*
in the code I have written there is no error, but when I want to run the project (same code as the file which have worked in maven before) this error appears !

please help me!
many thanks in advance.

**Comment by amjad-sahi** [comment]
@hamedisu,

Well, you have not provided the details about your gradle configurations, please provide us complete details about your gradle configurations and settings. We think most possible cause of your issue should be due to the configuration of your environment, you need to evaluate it and fix it. If you still could not evaluate, we need your sample project to test your issue on our end.

**Comment by hamedisu** [comment]
ok
many thanks for your guide.
I find out ... I should add   
`maven {
        url "https://artifact.aspose.com/repo/"
    }`
 at the repository section ...
thank you very much.

**Comment by amjad-sahi** [comment]
@hamedisu ,

Yes, you should add maven repository into your build.gradle. Anyways, we are pleased that you have sorted your issue now.
